src/AssembleMomentumEdgeSolverAlgorithm.C

```diff
@@ -414,6 +769,265 @@ AssembleMomentumEdgeSolverAlgorithm::execute()
   }
 }
 
+//--------------------------------------------------------------------------
```

Comment:
Is this a copy paste from the scalar implementation that I coded a few years back?